src/metabase/embedding/api.clj

```diff
@@ -14,3 +16,7 @@
 (def ^{:arglists '([request respond raise])} preview-embedding-routes
   "`/api/preview_embed` routes"
   (api.macros/ns-handler 'metabase.embedding.api.preview-embed))
+
+(def ^{:arglists '([request respond raise])} theme-routes
+  "`/api/embed-theme` routes"
```

Comment:
I checked and every other endpoint in metabase is dash-cased, only `preview_embed` is underscore_cased.